src/vs/workbench/contrib/preferences/browser/settingsEditor2.ts

```diff
@@ -538,6 +547,11 @@ export class SettingsEditor2 extends EditorPane {
 						return `@${LANGUAGE_SETTING_TAG}${languageId} `;
 					}).sort();
 					return sortedLanguages.filter(langFilter => !query.includes(langFilter));
+				} else if (queryParts[queryParts.length - 1].startsWith(`@${EXTENSION_SETTING_TAG}`)) {
+					const installedExtensionsTags = this.installedExtensionIds.map(extensionId => {
+						return `@${EXTENSION_SETTING_TAG}${extensionId} `;
+					}).sort();
+					return installedExtensionsTags.filter(extFilter => !query.includes(extFilter));
```

Comment:
I wonder whether we can show the "display name" of the extension in the suggest widget as well? Maybe you can return a "detail" or "description" to show next to the ID

Reply:
I'll look into returning completion items for that case.